src/boot/bootctl-install.c

```diff
@@ -449,7 +449,84 @@ static int copy_one_file(const char *esp_path, const char *name, bool force) {
         return ret;
 }
 
-static int install_binaries(const char *esp_path, const char *arch, bool force) {
+static int install_binaries_extra(const char *esp_path, const char *arch, bool force, char** ret_default_loader_path) {
+        char *root = IN_SET(arg_install_source, ARG_INSTALL_SOURCE_AUTO, ARG_INSTALL_SOURCE_IMAGE) ? arg_root : NULL;
+        _cleanup_closedir_ DIR *d = NULL;
+        _cleanup_free_ char *path = NULL;
+        _cleanup_free_ char *default_loader_path = NULL;
+        int r;
+
```

Comment:
can you add a comment here that explains how the source dir is expected to be set up, and where this is precisely copied?

Reply:
unaddressed